.editorconfig

```diff
@@ -6,5 +6,5 @@ indent_style = space
 indent_size = 2
 charset = utf-8
 trim_trailing_whitespace = true
-insert_final_newline = true
-end_of_line = lf
\ No newline at end of file
+insert_final_newline = false
```

Comment:
please, do not edit this file

Reply:
When this is set to true, the cursor in VsCode starts to jump to the next line.